platform/ui/src/gatsby-theme-docz/tailwind.css

```diff
@@ -1,3 +1,6 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+/* IMPORT CUSTOM FONT */
+@import url('https://fonts.googleapis.com/css?family=Lato&display=swap');
```

Comment:
Similar to current docz, let's add a docs page for font:

https://react.ohif.org/getting-started#usage